### ISSUE TITLE
any way to have multiple short base urls?

### ISSUE THREAD

**Issue by goellner** [ash-jc-allen/short-url#251]
I have set up a custom `default_url` in the config, now i need to override it, to generate a short link with a different base url than the one in the config. 

Tried overwriting the config and failed with this: 
```
config(['short-url.default_url' => 'https://link.asdf.com']);
```

Is there any way to override at runtime? 

full context of my example: 

```
$og_default_url = config('short-url.default_url');

// change default_url config to new one temporarily
config(['short-url.default_url' => config('app.deep_link_url')]);

$builder = new Builder();

$builder = $builder
    ->destinationUrl($destination_url_link)
    ->urlKey($url_key)
    ->redirectStatusCode(302);

// revert default_url config
config(['short-url.default_url' => $og_default_url]);
```

**Comment by ash-jc-allen** [comment]
Hey @goellner! Sorry to hear that you're having problems with the package. I've just tried running your code in the example and just want to check something.

I don't know if your example above is exactly the same as the code in your project, but in your example, you're not calling the `make` method on the `Builder` class. The Short URL won't be created until you've called that method.

Is it possible that you're calling `make` after you've switched your config back to the original values?

I've updated your example slightly and this seems to work:

```php
$og_default_url = config('short-url.default_url');

// change default_url config to new one temporarily
config(['short-url.default_url' => config('app.deep_link_url')]);

$builder = new Builder();

$shortUrl = $builder
    ->destinationUrl($destination_url_link)
    ->urlKey($url_key)
    ->redirectStatusCode(302)
    ->make();

// revert default_url config
config(['short-url.default_url' => $og_default_url]);

dd($shortUrl);
```

In the example above, I've just added `make` to the end of the chain. Does adding this on your end solve your issue? 😄
